content/app/development/ux/images/_index.en.md

```diff
@@ -66,24 +66,25 @@ The image can also have separate sources for different languages. The default so
 
 ## Hosting images from apps
 
-If the image should be loaded from the app, you need to set up static hosting of files in the application.
+If the image should be loaded from the app, you need to set up static hosting of files in the application (if you have an older app).
 This is configured in _App/Startup.cs_, in the _Configure_ method. This will host all files that is inside the `/app/wwwroot` folder. If this folder does not exist, it needs to be created.
 If you want to refer to the file `app/wwwroot/bilde_nb.png` it can be reached at the following relative url `/org/app-name/bilde_nb.png`
 
-Swap out _org/app-name_ with your organisation and app name. Example:
+_applicationId_ is a variable from a few lines above containing _org/app-name_ from `applicationmetadata.json`:
 
 ```C# {linenos=false,hl_lines=[5]}
 public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
   {
     // ...
     app.UseRouting();
-    app.UseStaticFiles("/org/app-name");
+    app.UseStaticFiles('/' + applicationId);
```

Comment:
Application developers are free to modify startup as they see fit, so I would prefer being explicit here. There is no guarantee that we don't update the variable name ourselves or move the swagger config elsewhere in the future. <

Reply:
So, how will the story on copying apps be then? For some apps we will need a new name every year in order to make some small changes. I'd prefer if `org/app-name` wasn't hard coded everywhere.

Reply:
Isn't this only an issue for older applications though? For new apps this configuration will already be in place and use the applicationId variable. Just thinking the doc should be explicit for those having to do the steps manually.

Reply:
I think encouraging to use variables for this makes sense, so would be good to keep the example like in this PR. For anyone needing to add this manually, it is explained a couple lines above what the variable contains - I think that should be enough.